frontend/src/scenes/pipeline/pipelineAccessLogic.tsx

```diff
@@ -0,0 +1,27 @@
+import { connect, kea, path, selectors } from 'kea'
+import { canConfigurePlugins, canGloballyManagePlugins } from 'scenes/plugins/access'
+import { teamLogic } from 'scenes/teamLogic'
+import { userLogic } from 'scenes/userLogic'
+
+import { AvailableFeature } from '~/types'
+
+import type { pipelineAccessLogicType } from './pipelineAccessLogicType'
+
+export const pipelineAccessLogic = kea<pipelineAccessLogicType>([
```

Comment:
We use various parts of it across transformations, destinations, newButton - in order to not load destinations, when not needed I moved this to a separate logic that doesn't load anything else besides user